### PR TITLE
fix: place receive at the top instead of middle

### PR DESCRIPTION
**Pull request by Tuditi**
## Summary

Upon receiving the generate address button is centered in the middle of the screen. This is moved to the top

### Changelog
```
move generate address to the top
```

## Relevant Issues

closes: https://github.com/iotaledger/firefly/issues/2690

## Type of Change
Please select any type below that is applicable to your changes, and delete those that are not.
- [ ] __Breaking__ - any change that would cause existing functionality to not work as expected
- [ ] __Chore__ - refactoring, build scripts or anything else that isn't user-facing
- [ ] __Docs__ - changes to the documentation
- [x] __Fix__ - a change which fixes an issue
- [ ] __New__ - a change which implements a new feature
- [ ] __Update__ - a change which updates existing functionality

### Platforms
Please select any platforms where your changes have been tested.
- __Desktop__
	- [ ] MacOS
	- [x] Linux
	- [ ] Windows
- __Mobile__
	- [ ] iOS
	- [ ] Android

### Instructions
Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

## Checklist
Please tick all of the following boxes that are relevant to your changes, and delete those that are not.
- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by jeeanribeiro — CHANGES_REQUESTED** [review]
@maxwellmattryan has a good point, if the button isn't at the bottom, the generated QR code pushes it, and I think it's a good UX to have buttons that don't change their position

File: packages/shared/routes/dashboard/wallet/views/Receive.svelte [hunk 1]
```diff
@@ -13,14 +13,15 @@
...
     const handleCloseClick = (): void => {
         $accountRouter.previous()
     }
 </script>
 
-<div class="w-full h-full space-y-6 flex flex-auto flex-col flex-shrink-0 p-6">
+<div class="w-full h-full space-y-6 flex flex-auto flex-col flex-shrink-0 p-8">
     <div class="w-full flex flex-row justify-between items-center">
-        <div class="w-full flex flex-row space-x-4 items-center">
+        <div class="w-full flex flex-row space-x-4 items-center justify-center">
```
Comment (maxwellmattryan) [inline packages/shared/routes/dashboard/wallet/views/Receive.svelte:24]: I think we should leave this to the left, as to match all of the other dashboard tiles.

Comment (Tuditi) [reply packages/shared/routes/dashboard/wallet/views/Receive.svelte:24]: Oops, good catch! I thought it looked off and that I had changed it to go to the left. Apparently not! :sweat_smile: . Will change it back!

**Review by maxwellmattryan — APPROVED** [review]
LGTM!

**Review by jeeanribeiro — APPROVED** [review]

**Pull request merged**